.github/workflows/integration-tests-vm-type.yml

```diff
@@ -23,11 +23,6 @@ on:
           Tag used for the test container
         type: string
         required: true
-      offline-mode:
-        description: |
-          Set to true to enable collector running in offline mode
-        type: boolean
-        default: false
```

Comment:
Is there a relation between the removal of `offline-mode` and the test summary ?

Reply:
> It also removes the unused COLLECTOR_OFFLINE_MODE env variable, only because I saw it while working on this.

I basically forgot to remove it as part of #1773 and stumbled across it while making these changes. I can move it to a separate PR if you'd like.

Reply:
It is fine (and in the same file :smile: )